Emotion/Android/ClientLibrary/lib/src/main/java/com/microsoft/projectoxford/emotion/contract/Scores.java

```diff
@@ -41,4 +41,23 @@
     public double neutral;
     public double sadness;
     public double surprise;
+    
+    public SortedMap<Double, String> ToRankedList(Order order)
```

Comment:
Please make the key the string, and the score (Double) the value.  Otherwise you would end up folding two emotions together if their scores were identical.